### PR TITLE
best-case one round

### PR DESCRIPTION
**Pull request by dsadas1212**

### CODE REVIEW

File: consensus/Reputation.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ const (
 	pEpsilonEqui  = 1.5
 	pEpsilonMali  = 1
 	vEpisilonMali = 1
-	gama          = 0.5
+	gama          = 0.01
```
Comment (SebastianElvis) [inline consensus/Reputation.go:16]:
```suggestion
	gamma          = 0.01
```

File: consensus/protocol.go [hunk 2]
```diff
@@ -42,23 +44,33 @@ func (shs *SyncHS) Init(c *config.NodeConfig) {
...
 	shs.cmdChannel = make(chan []byte, ProtocolMsgBuffer)
 	shs.voteChannel = make(chan *msg.Vote, ProtocolMsgBuffer)
+	shs.blockCandidateChannel = make(chan *chain.Candidateblock, ProtocolMsgBuffer)
 
 	shs.certMap = make(map[uint64]*msg.BlockCertificate)
 	// Setup certificate for the first block
 	shs.certMap[0] = &msg.GenesisCert
+
+	shs.callFuncNotFinish = true
+	shs.gcallFuncFinish = true
```
Comment (SebastianElvis) [inline consensus/protocol.go:68]: What do variables at L67-68 mean?

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,7 @@
 client/client
 node/node
-*.log
\ No newline at end of file
+*.log
+quicktest.sh
```
Comment (SebastianElvis) [inline .gitignore:4]: Why do we need to ignore this?

**Review by SebastianElvis — APPROVED** [review]

File: consensus/protocol.go (hunk 2 above)
Comment (dsadas1212) [reply consensus/protocol.go:68]: There's something wrong with my initial description above,

The goal is to prevent the timer from constantly turning on

**Pull request merged**